setup.py

```diff
@@ -65,9 +65,15 @@ def run(self):
 			shutil.move(file, renamed_file)
 
 			if sys.platform == "win32":
+				py = sys.executable
+				pyw = py.replace('.exe', 'w.exe')
+				pyw_found = isfile(pyw)
+				if pyw_found:
+					py = pyw
+
 				# Generate a batch script to wrap the python script so we could invoke
 				# that script directly from the command line
-				batch_script = "@echo off\n\"%s\" \"%s\" %%*" % (sys.executable, renamed_file)
+				batch_script = "@echo off\n%s\"%s\" \"%s\" %%*" % ("start \"\" " if pyw_found else "", py, renamed_file)
```

Comment:
Can you please change this line to use single quotes literals, so that escaping is not needed? I.e.
```python
batch_script = '@echo off\n%s"%s" "%s" %%*' % ('start "" ' if pyw_found else '', py, renamed_file)
```
which looks much more readable than what you currently have.